README.md

```diff
@@ -27,10 +27,26 @@ tracker](https://github.com/share/sharedb/issues).
 - Projections to select desired fields from documents and operations
 - Middleware for implementing access control and custom extensions
 - Ideal for use in browsers or on the server
-- Reconnection of document and query subscriptions
 - Offline change syncing upon reconnection
```

Comment:
@mbalex99 I mean, if reconnection doesn't come out of the box, I think this line should also be removed, as it's misleading.